src/main/scala/firrtl/passes/RemoveAccesses.scala

```diff
@@ -54,7 +54,7 @@ object RemoveAccesses extends Pass {
       ls.zipWithIndex map {case (l, i) =>
         val c = (i / stride) % wrap
         val basex = l.base
-        val guardx = AND(l.guard,EQV(uint(c),e.index))
+        val guardx = AND(l.guard,EQV(UIntLiteral(c),e.index))
```

Comment:
This is fine but I feel compelled to voice my distaste for the lower case L as a variable name

Reply:
I totally agree, obviously this PR was not trying to do general style improvements, but I think some effort in that direction would be a good idea. Maybe I'll work on getting IntelliJ up and running